recipe/meta.yaml

```diff
@@ -1,47 +1,58 @@
 {% set name = "jsonpath-ng" %}
-{% set version = "1.5.3" %}
+{% set version = "1.6.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a273b182a82c1256daab86a313b937059261b5c5f8c4fa3fc38b882b344dd567
+  sha256: 086c37ba4917304850bd837aeab806670224d3f038fe2833ff593a672ef0a5fa
 
 build:
-  noarch: python
+  skip: true #[py<37]
   number: 0
   entry_points:
     - jsonpath.py=jsonpath_ng.bin.jsonpath:entry_point
-  script: {{ PYTHON }} -m pip install --no-deps . -vv
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - wheel
+    - setuptools >=18.5
   run:
-    - python >=3.6
+    - python
     - ply
     - decorator
     - six
```

Comment:
```suggestion
```
`six` was removed, see https://github.com/h2non/jsonpath-ng/commit/b93282a25943d8c22539bf0a9960a01e6c8a6fba#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7.
`decorator` is not needed, see https://github.com/h2non/jsonpath-ng/blob/bf75ddf25a552f58aca4f2f36cd69d21b4727c76/History.md?plain=1#L66